robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -154,14 +154,27 @@ static void rq_state_start_stream()
  */
 static void rq_state_run()
 {
+	static unsigned char retries = 0;
+
 	rq_com_listen_stream();
 
 	if(rq_com_get_valid_stream() == false)
 	{
+		retries++;
+		printf("invalid stream : retry #%d\n", retries);
+
+		if (retries >= 100)
```

Comment:
Is there a best practice how to resolve such an issue? This part is somehow unrelated to ros and probably should not include ros.h just to use the logging channel.
I would remove it for now, but the general question stays the same